desk/sur/groups.hoon

```diff
@@ -273,6 +273,57 @@
 ::  $flagged-content: flagged posts and replies that need admin review
 ::
 +$  flagged-content  (map nest (jug plan ship))
+::
++$  request-id  @uv
++$  action
+  $:  =request-id
+      =a-groups
+  ==
++$  command
+  $:  =request-id
+      =c-groups
+  ==
++$  response
+  $:  id=request-id
+      body=response-body
+  ==
++$  response-update
+  $:  id=request-id
+      body=response-update-body
+  ==
++$  response-body
+  $%  [%ok =r-group]
+      [%error type=action-error message=tang]
+      [%pending status=poke-status]
+  ==
++$  response-update-body
+  $%  [%ok =u-group]
+      [%error type=action-error message=tang]
+  ==
+::
++$  requests  (map request-id incoming-request)
+::  $incoming-request: a request coming into the agent
+::
+::  .id: request id
+::  .http-id: http request id, if any. means we should give a response
```

Comment:
This comment line is a bit confusing: _what_ means we should give a response? If request-id is present?